teuthology/provision/pelagos.py

```diff
@@ -71,18 +71,22 @@ def __init__(self, name, os_type, os_version=""):
             self.os_name = os_type
         self.log = log.getChild(self.name)
 
-    def create(self):
+    def create(self, init_only=False):
```

Comment:
Could you please add the description of the init_only parameter to the __doc__ for this method.

Reply:
Fixed